fdbclient/NativeAPI.actor.cpp

```diff
@@ -3693,12 +3733,17 @@ void Transaction::setOption( FDBTransactionOptions::Option option, Optional<Stri
 			span.addParent(BinaryReader::fromStringRef<UID>(value.get(), Unversioned()));
 			break;
 
-	    case FDBTransactionOptions::REPORT_CONFLICTING_KEYS:
-		    validateOptionValue(value, false);
-		    options.reportConflictingKeys = true;
-		    break;
+		case FDBTransactionOptions::REPORT_CONFLICTING_KEYS:
```

Comment:
I assume it's clang-format's decision to use tab?

Reply:
Yes. In the old file, there are several lines that remain unaligned. Just want to reformat them here.